custom_components/tahoma/climate_devices/atlantic_heat_recovery_ventilation.py

```diff
@@ -0,0 +1,193 @@
+"""Support for AtlanticHeatRecoveryVentilation."""
+import logging
+from typing import List, Optional
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    FAN_AUTO,
+    HVAC_MODE_FAN_ONLY,
+    SUPPORT_FAN_MODE,
+    SUPPORT_PRESET_MODE,
+)
+from homeassistant.const import TEMP_CELSIUS
+
+from ..coordinator import OverkizDataUpdateCoordinator
+from ..entity import OverkizEntity
+
+FAN_BOOST = "home_boost"
+FAN_KITCHEN = "kitchen_boost"
+FAN_AWAY = "away"
+FAN_BYPASS = "bypass_boost"
+
+PRESET_AUTO = "auto"
+PRESET_PROG = "prog"
+PRESET_MANUAL = "manual"
+
+OVERKIZ_TO_FAN_MODES = {
+    OverkizCommandParam.AUTO: FAN_AUTO,
+    OverkizCommandParam.AWAY: FAN_BOOST,
+    OverkizCommandParam.BOOST: FAN_KITCHEN,
+    OverkizCommandParam.HIGH: FAN_AWAY,
```

Comment:
The Overkiz fan modes seems to not match with the command fan modes

Should it be
```
    OverkizCommandParam.AUTO: FAN_AUTO,
    OverkizCommandParam.AWAY: FAN_AWAY,
    OverkizCommandParam.BOOST: FAN_BOOST,
    OverkizCommandParam.HIGH: FAN_KITCHEN,
``` 
?

Reply:
Good catch. Could you try the latest version?